people.md

```diff
@@ -0,0 +1,29 @@
+# pandas People
+
+## Benevolent Dictator for Life
+
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
+
+## Project Core Developers
+
+- Tom Augspurger
+- Phillip Cloud
+- Andy Hayden
+- Stephan Hoyer
+- Thomas Kluyver
+- Wes McKinney
+- Wouter Overmeire
```

Comment:
We should evaluate the Core Team in line with the governance doc (i.e. trimming inactive devs) but for now we'll bring along everyone